lib/errorHandlers/suppressError.ts

```diff
@@ -15,25 +15,43 @@ const {
 
 const i18nKey = 'lib.errorHandlers.suppressErrors';
 
-function createPlatformVersionError(err, subCategory) {
+function createPlatformVersionError(err: unknown, subCategory: string): void {
   let translationKey = 'unspecifiedPlatformVersion';
   let platformVersion = 'unspecified platformVersion';
   const errorContext =
-    err.response && err.response.data && err.response.data.context;
+    typeof err === 'object' &&
+    err !== null &&
+    'response' in err &&
+    typeof err.response === 'object' &&
+    err.response !== null &&
+    'data' in err.response &&
+    typeof err.response.data === 'object' &&
+    err.response.data !== null &&
+    'context' in err.response.data
+      ? err.response.data.context
+      : undefined;
 
   switch (subCategory) {
-    case [PLATFORM_VERSION_ERROR_TYPES.PLATFORM_VERSION_RETIRED]:
+    case PLATFORM_VERSION_ERROR_TYPES.PLATFORM_VERSION_RETIRED:
       translationKey = 'platformVersionRetired';
-      if (errorContext && errorContext[subCategory]) {
-        platformVersion = errorContext[subCategory];
+      if (
+        errorContext &&
+        (errorContext as { [key: string]: string })[subCategory]
```

Comment:
I think if you sort out the above issue with `HubspotHttpError` you shouldn't have to cast here

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1251/commits/146dcbafeabd6d865cb305cb61cf867207c31fbf